src/components/ArticleList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react"
+import { fetchArticleData } from "./api"
+import ArticleCard from "./ArticleCard"
+import DisplayBox from "./DisplayBox"
+import { Link } from "react-router-dom"
+
+function ArticleList() {
+
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        fetchArticleData().then((articleData) => {
+            setArticles(articleData.data.articles)
+        })
+        .catch((err) => console.log(err))
+    }, [])
+
+    return (
+        <section id="article-list">
+            <h2>Articles</h2>
+            <ul id="article-list">
+                {articles.map((article) => {
+                    return (
+                        <>
```

Comment:
If you used something other than a react fragment here, you could pass key in as a prop and not have to do it twice below.